### DISCUSSION TITLE
No replies from OpenAI for WhatsApp template

### DISCUSSION THREAD

**Discussion by kmanan** [AntlerVC/firetable#1222] [Support / Q&A] [answered]
Hi, I'm trying to use the WhatsApp template and I think I followed all instructions I am not getting any replies.

The reply column is blank 
![image](https://user-images.githubusercontent.com/17659155/233793311-6e154a78-97fc-48ca-8c54-91343706cac3.png)

EDIT: I deleted and recreated the bot with the new secret that has both SID and auth key. Same result. I can see the messages being sent but there is no reply

**Comment by kmanan** [comment]
Seeing this error in Twilio logs
![image](https://user-images.githubusercontent.com/17659155/233808373-40f9a5fc-7b0a-4712-9933-046978775222.png)

**Comment by BeeBombshell** [comment, marked as answer]
Hey @kmanan, from the above screenshot, I see that your Reply derivative column is not generated. Have you successfully set up your OpenAI Secret? Could you go to the GCP Cloud Logs to check in case of a specific error: https://console.cloud.google.com/logs/ ?

**Reply by kmanan** [reply]
Thanks for the response. I had a call with one of the engineers and we narrowed down a few bugs in the code. I am still not getting the responses, the engineer has access to my account and is investigating.

**Reply by BeeBombshell** [reply]
Sounds good, @kmanan. Hope your issue gets resolved soon! If you have any other issues or queries feel free to reach out.
